### PR TITLE
 Add navbar

### PR DESCRIPTION
**Pull request by jhaanupama**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
-
+import React from 'react';
 import './App.css'; 
 import Navbar from "./Navbar";
 import Footer from "./Footer";
-
+import Submit from "./Submit";
+import {BrowserRouter as Router,Route, Routes, BrowserRouter} from 'react-router-dom';
 import MainContent from "./MainContent";
+import { Switch } from 'react-router-dom/cjs/react-router-dom.min';
```
Comment (shawshankkumar) [inline src/App.jsx:8]: iska kya use hai?

File: src/Navbar.jsx [hunk 2]
```diff
@@ -4,7 +4,11 @@ function Navbar() {
   return (
     <div className='Navbar'>
       <h1>The Recipe Stash</h1>
-      <p>This is the navbar</p>
+      <a href="/">Home</a>
+      <a href="/Submit">Submit Recipie</a>
```
Comment (shawshankkumar) [inline src/Navbar.jsx:8]: routes ka names are always /submit and /about hona chaiye

Comment (shawshankkumar) [reply src/Navbar.jsx:8]: File name bada hota, url chota.

**Review by shawshankkumar — CHANGES_REQUESTED** [review]
Also format, cntrl shift p and then format using prettier

File: src/App.jsx (hunk 1 above)
Comment (jhaanupama) [reply src/App.jsx:8]: react-router-dom is a standard library used for routing among various components in react.
switch component looks through all child nodes and gives the first path that matches url.

**Comment by shawshankkumar** [comment]
but do we really need to import it from cjs? direct ni?

**Pull request merged**